images/httpbin/rootfs/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.16.1
+FROM alpine:3.16.2
```

Comment:
I think it will be failed. 

But we can go on.  There is now a PR to replace it with httpbun

Reply:
yes, it will fail. And yes we can go on.